### ISSUE TITLE
Customize the Landing Page

### ISSUE THREAD

**Issue by 0xCipher-max** [Creatoon/Jar-web-app#32]
The landing page which comes after logging in can be made more attractive and interesting by adding animations to the buttons and adding other effects and adding a footer.

**Comment by 0xCipher-max** [comment]
I can definitely make it better

**Comment by Creatoon** [comment]
@0xCipher-max Great to see your enthusiasm, but it will be great if you think about the main landing page.
For example, If you visit https://jaarr.herokuapp.com , then you get redirected to the signup page, instead of redirecting to a signup page, we should redirect to a landing page(which is not being made yet), so if you make a landing page for the website, then it would be great.

Like the landing page should contain logo, nice description about our work, CTA buttons (signup and login), a nice footer with terms, privacy. 

Tou can implement what you are saying also, it's all up to you.

Thanks

**Comment by varunwho** [comment]
@Creatoon yes it can definitely be made better.

**Comment by 0xCipher-max** [comment]
Yes I can create a Main Landing Page as well Please assign me this issue

**Comment by Creatoon** [comment]
@0xCipher-max go ahead, kindly inform me if you find any issue.
Just make the color scheme same, like the blue color we used in the webpage, you can get that from inspecting the page.

**Comment by 0xCipher-max** [comment]
> @0xCipher-max go ahead, kindly inform me if you find any issue.
> Just make the color scheme same, like the blue color we used in the webpage, you can get that from inspecting the page.

Hi there I have created the Template and linked it with the server. I need you to check the template and give me some details to add in About and ContactUs Page

**Comment by Creatoon** [comment]
I cant see any pr made by you for me to check.

**Comment by ashuydv** [comment]
can i work on this ??

**Comment by 0xCipher-max** [comment]
> I cant see any pr made by you for me to check.

Please Check now

**Comment by 0xCipher-max** [comment]
I haven't made a direct redirection to the homepage now I can do it after you approve the design
